harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -1022,6 +1118,17 @@ Suggest:
 
 
 
+Lint:    Miscellaneous (0 priority)
+Message: |
+     679 | another snatch in the air. This time there were two little shrieks, and more
+     680 | sounds of broken glass. “What a number of cucumber-frames there must be!”
+         |                  ^~~~~ Consider adding an article before this noun.
```

Comment:
This reveals two problems. We've already covered that "glass" is both a mass noun and countable noun. Here it's a mass noun.

But even if it were only countable and needed correction, the article would actually belong before the adjective that precedes the noun: "of a broken glass", not "of broken a glass".